pkg/resource/k8s-events-config/events-logger-config.go

```diff
@@ -0,0 +1,61 @@
+package k8seventsconfig
+
+import (
+	"fmt"
+
+	"github.com/blang/semver"
+	"github.com/pkg/errors"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/logging-operator/pkg/common"
+	loggedcluster "github.com/giantswarm/logging-operator/pkg/logged-cluster"
+)
+
+const eventsLogggerConfigName = "events-logger-config"
+
+func GenerateEventsLoggerConfig(lc loggedcluster.Interface, observabilityBundleVersion semver.Version, defaultNamespaces []string) (v1.ConfigMap, error) {
+	var values string
+	var err error
+
+	switch lc.GetKubeEventsLogger() {
+	case common.EventsLoggerGrafanaAgent:
+		values, err = GenerateGrafanaAgentConfig(lc, defaultNamespaces)
```

Comment:
Careful about the cm name here, same for secrets

Reply:
Took care of it.